### PR TITLE
Add document types end point

### PR DESCRIPTION
**Pull request by KyleMacPherson**
### What?
Add an end point to render all unique document types.

[Trello card](https://trello.com/c/7zjqCznt/726-2-index-page-filter-by-document-type)

### Why?
So that we can render a dropdown with all document types in our front end to allow users to filter by document type

screen shot 2018-10-12 at 13 06 30

### CODE REVIEW

**Comment by KyleMacPherson** [comment]
[(2) Index page - filter by document type](https://trello.com/c/7zjqCznt/726-2-index-page-filter-by-document-type)

**Comment by theseanything** [comment]
Maybe include a test, so we know are getting the appropiate document types from the DB i.e. only the latest?

File: app/domain/queries/find_all_document_types.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class Queries::FindAllDocumentTypes
```
Comment (pmanrubia) [inline app/domain/queries/find_all_document_types.rb]: We would need a Unit test for this class

File: spec/domain/queries/find_all_document_types_spec.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+RSpec.describe Queries::FindAllDocumentTypes do
+  context 'when we have data' do
+    before do
+      create :user
+      create :edition, document_type: 'guide'
+      create :edition, document_type: 'travel_advice'
+      create :edition, document_type: 'manual'
+    end
+
+    it 'returns distinct organisations ordered by primary_organisation_title' do
```
Comment (pmanrubia) [inline spec/domain/queries/find_all_document_types_spec.rb]: What about adding an edition that is not latest and checking that it returns the correct value?

**Pull request merged**